src/libPMacc/include/pmacc_types.hpp

```diff
@@ -176,16 +178,14 @@ enum EventType
  * you must align all array and structs which can used on device
  * @param byte byte of data which must aligned
```

Comment:
You could also update this. "can be used on device" and "size of data in bytes" for better understanding